linode/manager
# feat: [M3-6427] - Add No Results section for Marketplace Search

## Description 📝
Add No Results section for Marketplace Search

## Preview 📷
![image](https://user-images.githubusercontent.com/119517080/232818144-ee181b17-3a4b-4f67-8aa4-9adec4cbaa5a.png)


**Remove this section or include a screenshot or screen recording of the change**

## How to test 🧪

- Navigate to Marketplace search
- Search for app that doesn't exists (ex; datadog)
- Should display "No Results" message. 

**How do I run relevant unit or e2e tests?**

- yarn test
- yarn cy:run
